sympy/solvers/tests/test_solveset.py

```diff
@@ -2315,5 +2315,5 @@ def test_solve_modular_fail():
 # end of modular tests
 
 def test_issue_17276():
-    assert nonlinsolve([Eq(x,5**1/5), Eq(x*y, 25*sqrt(5))], x, y) == \
-    FiniteSet((1.0, 25.0*sqrt(5)))
+    assert nonlinsolve([Eq(x, 5**(Integer(S(1))/5)), Eq(x*y, 25*sqrt(5))], x, y) == \
```

Comment:
`S(1)` is the same as `Integer(1)` so `S(1)` alone will suffice and it is not necessary to import `Integer`.

Reply:
I wonder is using `Integer(1)` is the right practice while using sympy methods we should use `sympify` (S(1)) .just asking that it maybe usefull for me in future while working with sympy.